src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -56,6 +57,20 @@ public NodeReadConverter(CustomNodeManager manager, LibraryServices libraryServi
             }
         }
 
+        public NodeReadConverter(CustomNodeManager manager, LibraryServices libraryServices, NodeFactory nodeFactory, bool isTestMode = false)
+        {
+            this.manager = manager;
+            this.libraryServices = libraryServices;
+            this.nodeFactory = nodeFactory;
+            this.isTestMode = isTestMode;
```

Comment:
Minor but I think we avoid using `this` as per existing coding conventions (which may have been forgotten along the way). Also if you use Resharper you'll see that a lot of these things are highlighted by the tool.